string message
---
int32 toTurn
